cli/templates/structure/jsonnet-files/index.md

```diff
@@ -98,6 +98,13 @@ Example:
   - Returns `keboola.wr-db-snowflake` for AWS stacks.
   - Returns `keboola.wr-snowflake-blob-storage` for Azure stacks.
 
+--------------------------------------
+
+**`HasProjectBackend(backends []string) bool`**
```

Comment:
Here should be `HasProjectBackend(backend string) bool` - only one backend.
- The condition can be used multiple times, but at 99% of cases it will be used once.
- I will add comment also to the code review.